Dockerfile.dapper

```diff
@@ -1,14 +1,14 @@
-FROM golang:1.14.7
+FROM golang:1.16
```

Comment:
Should we pin to 1.16.3?

Reply:
Yeah, probably, I was following Darren here https://github.com/rancher/rancher/blob/master/Dockerfile.dapper#L28 😄